linetools/lists/sets/llist_v1.1.ascii

```diff
@@ -1121,7 +1121,7 @@
 | 1200.7098 |    NI 1200c |    1 |   1 |   0 |   0 |   0 |    0 |   0 |   0 |    0 |
```

Comment:
Changes to this file are unrelated to this PR. Need @profxj to assess whether this is harmful or not.